shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -176,18 +177,11 @@ static BOOL DoesHardwareSupportWideGamut() {
   settings.enable_wide_gamut = enableWideGamut;
 #endif
 
-  if (!command_line.HasOption("enable-impeller")) {
-    // Next, look in the app bundle.
-    NSNumber* enableImpeller = [bundle objectForInfoDictionaryKey:@"FLTEnableImpeller"];
-    if (enableImpeller == nil) {
-      // If it isn't in the app bundle, look in the main bundle.
-      enableImpeller = [mainBundle objectForInfoDictionaryKey:@"FLTEnableImpeller"];
-    }
-    // Change the default only if the option is present.
-    if (enableImpeller != nil) {
-      settings.enable_impeller = enableImpeller.boolValue;
-    }
-  }
+#if FML_OS_IOS_SIMULATOR
+  // Fallback to Skia which itself falls back to software rendering on simulators that cannot
```

Comment:
FYI @zanderso what is the plan with the no-op backend?

I think we need to respect the enable impeller setting here and use the no-op backend if there is no mtl device. otherwise we're actually forcing folks to use skia that may have already opted in

Reply:
Yeah, I hadn't considered that folks might have already removed the opt-out who are on simulator without Metal. Ignore the flag and making them use Skia would be the opposite of what we're overall trying to do. Simulator should just continue to respect the flag for now. (But we should still remove the unit tests that explicitly try to target the software backend, I think?)

Reply:
Yes, see: https://github.com/flutter/engine/blob/main/shell/platform/darwin/ios/ios_surface.mm#L19